cs/src/comm/epoxy-transport/Frame.cs

```diff
@@ -48,10 +50,12 @@ public Framelet(FrameletType type, ArraySegment<byte> contents)
 
         public static bool IsKnownType(UInt16 value)
         {
-            return value == (UInt16)FrameletType.LayerData
-                | value == (UInt16)FrameletType.PayloadData
-                | value == (UInt16)FrameletType.EpoxyHeaders
+            return false
```

Comment:
Is the false just to line up all the other checks? What's with all the single pipes (rather than double)?